pym/bob/cmds/build.py

```diff
@@ -947,7 +953,10 @@ def _cookPackageStep(self, packageStep, checkoutOnly, depth):
                         workspaceChanged = True
                         wasDownloaded = True
                     elif self.__forcedDownload:
-                        raise BuildError("Downloading artifact failed")
+                        if self.__forcedDownloadFallback:
+                            self.__forcedDownloadFallback = False
```

Comment:
This won't work if multiple packages are passed on the command line because after the first one `__forcedDownloadFallback` may have been reset.